pkg/controller/multiclusterobservability/multiclusterobservability_controller.go

```diff
@@ -385,6 +385,12 @@ func (r *ReconcileMultiClusterObservability) UpdateStatus(
 	return nil, nil
 }
 
+// labelsForMultiClusterMonitoring returns the labels for selecting the resources
+// belonging to the given MultiClusterObservability CR name.
+func labelsForMultiClusterMonitoring(name string) map[string]string {
+	return map[string]string{"observability.open-cluster-management.io/name": name}
+}
+
```

Comment:
What is it for? I did not find where to call it?

Reply:
this is dead code, I will remove it. thanks.